qa/suites/powercycle/osd/whitelist_health.yaml

```diff
@@ -1,4 +1,5 @@
-overrides:
-  ceph:
-    log-whitelist:
+ overrides:
+   ceph:
+     log-whitelist:
```

Comment:
Why the extra space? Not understanding "Not sure when thsi appeared in luminous, but it's missing from master."

Reply:
der, i my git checkout was just out of date